e2e_test/sink/remote/jdbc.check.pg.slt

```diff
@@ -3,17 +3,17 @@
 query I
 select * from t_remote_0 order by id;
 ----
-1 Alex 28208 281620391 4986480304337356659 28162.0391 2.03 28162.0391 2023-03-20 10:18:30
```

Comment:
Any reason to change from a normal datetime to a strange one?

Reply:
> Would you mind adding e2e test to cover the case of https://github.com/risingwavelabs/risingwave/issues/14198 in e2e_test/sink/remote/jdbc.load.slt?

cover the case of #14198.